### PR TITLE
gcfscape: Add version 1.8.6

### PR DESCRIPTION
**Pull request by ghost-420**

### CODE REVIEW

File: bucket/gcfscape.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.8.6",
+    "description": "A archive viewer and extractor for BSP, GCF, NCF, PAK, SGA, VPK, WAD, and XZP files.",
+    "homepage": "http://nemesis.thewavelength.net/index.php?c=169",
```
Comment (Ash258) [inline bucket/gcfscape.json:4]: image

File: bucket/gcfscape.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.8.6",
+    "description": "A archive viewer and extractor for BSP, GCF, NCF, PAK, SGA, VPK, WAD, and XZP files.",
+    "homepage": "http://nemesis.thewavelength.net/index.php?c=169",
+    "license": "Freeware",
```
Comment (Ash258) [inline bucket/gcfscape.json:5]:
```suggestion
    "license": "Freeware",
    "suggest": {
        "vcredist": "extras/vcredist2008"
    },
```

File: bucket/gcfscape.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.8.6",
+    "description": "A archive viewer and extractor for BSP, GCF, NCF, PAK, SGA, VPK, WAD, and XZP files.",
+    "homepage": "http://nemesis.thewavelength.net/index.php?c=169",
+    "license": "Freeware",
+    "url": "https://valvedev.info/tools/gcfscape/gcfscape.zip",
+    "hash": "5b61408a83b61afc9c775534cf02749edfde8e570bf17604bc191f6dc57055b0",
```
Comment (Ash258) [inline bucket/gcfscape.json:7]:
```suggestion
    "hash": "5b61408a83b61afc9c775534cf02749edfde8e570bf17604bc191f6dc57055b0",
    "architecture": {
        "64bit": {
            "extract_dir": "x64"
        },
        "32bit": {
            "extract_dir": "x86"
        }
    },
```

File: bucket/gcfscape.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        ]
+    ],
+    "architecture": {
+        "64bit": {
+            "extract_dir": "x64"
+        },
+        "32bit": {
+            "extract_dir": "x86"
+        }
+    }
```
Comment (Ash258) [inline bucket/gcfscape.json:25]:
```suggestion
    ]
```

File: bucket/gcfscape.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    "version": "1.8.6",
+    "description": "A archive viewer and extractor for BSP, GCF, NCF, PAK, SGA, VPK, WAD, and XZP files.",
+    "homepage": "http://nemesis.thewavelength.net/index.php?c=169",
+    "license": "Freeware",
+    "url": "https://valvedev.info/tools/gcfscape/gcfscape.zip",
+    "hash": "5b61408a83b61afc9c775534cf02749edfde8e570bf17604bc191f6dc57055b0",
+    "bin": "GCFScape.exe",
+    "suggest": {
+        "vcredist": "extras/vcredist2008"
+    },
```
Comment (Ash258) [inline bucket/gcfscape.json:11]:
```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/gcfscape.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.8.6",
+    "description": "A archive viewer and extractor for BSP, GCF, NCF, PAK, SGA, VPK, WAD, and XZP files.",
```
Comment (Ash258) [inline bucket/gcfscape.json:3]:
```suggestion
    "description": "An archive viewer and extractor for BSP, GCF, NCF, PAK, SGA, VPK, WAD, and XZP files",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by ghost-420** [comment]
csrss_thIpQjicka
EmuHawk_TSK1SpPLdP
What makes EmuHawk exempt?

**Comment by ghost-420** [comment]
alacritty_h51ARdXWXi

**Comment by Ash258** [comment]
How is EmuHawk even relevant to this conversation??

PR will be closed unless you provide valid, English homepage

**Comment by ghost-420** [comment]
> How is EmuHawk even relevant to this conversation??

Nevermind, I thought it was removed, not moved up.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**